packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,13 +1,36 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import { PaperPlane } from '@strapi/icons';
+
+import { pluginId } from './pluginId';
+
+import type { Plugin } from '@strapi/types';
 
 // eslint-disable-next-line import/no-default-export
-export default {
-  register() {
+const admin: Plugin.Config.AdminInput = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register(app: any) {
     if (window.strapi.features.isEnabled('cms-content-releases')) {
       // EE Code would be here
       // For testing if the plugin is working, add a console.log
```

Comment:
Lets remove these two comments now that we are doing stuff inside this conditional